docs/api-reference/extensions/v1beta1/operations.md

```diff
@@ -5,6 +5,9 @@
 <!-- needed for gh-pages to render html files when imported -->
 {% include operations.html %}
 
+<strong>
+The latest 1.0.x release of this document can be found
+[here](http://releases.k8s.io/release-1.0/docs/api-reference/extensions/v1beta1/operations.md).
```

Comment:
according to my commit message, it's added by hack/update-generated-docs.sh. But it's not added back this time after I manually remove it.